### PR TITLE
Added navigation

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
Please review
Screenshot (40)

### CODE REVIEW

**Comment by poojakelkar** [comment]
Good to merge

File: Navigation/nav.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
+:root {
+    --primary-color: #1d4ed8;
```
Comment (bharati-21) [inline Navigation/nav.css]: If these variables are common to all components, I suggest adding them to the common utility stylesheet.

File: Navigation/nav.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (bharati-21) [inline Navigation/nav.html:6]: You can add the meta author and description tags as well.

**Review by bharati-21 — APPROVED** [review]
Looks good to merge.

File: Navigation/nav.css (hunk 1 above)
Comment (syed-shaida-hussain) [reply Navigation/nav.css]: Done. Thank you.

**Pull request merged**